src/navigation/context.py

```diff
@@ -57,7 +64,7 @@ class Environment:
     ctx: Context
     NO_FIDUCIAL: ClassVar[int] = -1
 
-    def get_fid_pos(self, fid_id: int, frame: str = "map") -> Optional[np.ndarray]:
+    def get_fid_pos(self, fid_id: int, frame: str = "odom") -> Optional[np.ndarray]:
```

Comment:
Can we switch all uses of `get_fid_pos` to have an explicit argument for the frame, so it automatically changes when we change the configured param `use_odom`?